olm/update-resources.sh

```diff
@@ -173,6 +173,9 @@ updateNighltyBundle() {
       incrementNightlyVersion "${platform}"
     fi
 
+    cp -f "${ROOT_PROJECT_DIR}/deploy/crds/org.eclipse.che_checlusterbackups_crd.yaml" "${NIGHTLY_BUNDLE_PATH}/manifests"
+    cp -f "${ROOT_PROJECT_DIR}/deploy/crds/org.eclipse.che_checlusterrestores_crd.yaml" "${NIGHTLY_BUNDLE_PATH}/manifests"
```

Comment:
1. crd generation should be done in `generateCRD` function
2. I believe references to crds must be added into `csv-config.yaml` files

Reply:
3. Test files existence during release https://github.com/eclipse-che/che-operator/blob/main/make-release.sh#L235-L236
4. Update `olm/check-resources.sh` to check if crds are up to date like here https://github.com/eclipse-che/che-operator/blob/main/.github/bin/check-resources.sh#L52
5. Update `olm/prepare-community-operator.sh` to copy crds to PR like here https://github.com/eclipse-che/che-operator/blob/main/olm/prepare-community-operators-update.sh#L123-L126
6. Update `release-olm-files.sh` to create CRDs diffs on release

Reply:
Thank you for the full list of places where I should patch the scripts